src/main/java/com/minecolonies/coremod/entity/pathfinding/MinecoloniesAdvancedPathNavigate.java

```diff
@@ -330,6 +325,13 @@ else if (world.getBlockState(ourEntity.getPosition()).getBlock() == Blocks.GRASS
 		}
     }
 
+	public Block findBlockTypeUnderEntity(Entity parEntity)
```

Comment:
looks good, but you can make it private and please add javadoc and fix formatting